### PR TITLE
README "About Us" and Quick Start Link

### PR DESCRIPTION
**Pull request by jacknagz**
## Background

Add a small company manifesto to the README

## Changes
* Add an "About Us" section
* Small misc edits

## Testing
* Viewed in Atom

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,23 +21,32 @@
...
-- **SIEM:** Centralize all security log data for threat detection, historical search, long-term storage, and investigations
-- **[Log Analysis](https://runpanther.io/log-analysis):** Detect suspicious activity quickly and effectively with Python rules
-- **[Cloud Compliance](https://runpanther.io/compliance/):** Detect and enforce AWS infrastructure best practices with Python policies
+Panther is the destination for all security data to enable threat detection, compliance, historical search, and security investigations. Panther's core features include:
+
+- **[Log Analysis](https://runpanther.io/log-analysis):** Real-time detection of suspicious activity with Python rules
+- **[Compliance](https://runpanther.io/compliance/):** Real-time monitoring and enforcement of AWS infrastructure best practices with Python policies
 - **Alerting:** Send notifications to your team when new issues are identified
-- **Automatic Remediation:** Correct insecure infrastructure as soon as new issues are identified
+- **Automatic Remediation:** Correct insecure infrastructure as soon as possible
```
Comment (austinbyers) [inline README.md]: The "about" + "features" + "use cases" is getting a bit long-winded and redundant, in my opinion. It takes up an entire page of screen real estate.

Consider combining "product" + "use cases" or removing some bullet points. For example, "notifying your team" is in the "product" section as well as "alerting" here in the "use cases" section.

File: README.md [hunk 2]
```diff
@@ -60,10 +70,12 @@ Follow our [Quick Start Guide](https://docs.runpanther.io/quick-start) to deploy
 
 ## Contributing
 
-We love contributions! Please read the [contributing guidelines](https://github.com/panther-labs/panther/blob/master/docs/CONTRIBUTING.md) before submitting pull requests.
+We welcome all contributions! Please read the [contributing guidelines](https://github.com/panther-labs/panther/blob/master/docs/CONTRIBUTING.md) before submitting pull requests.
```
Comment (austinbyers) [inline README.md]: I think just "we welcome contributions" is sufficient, but no strong preference.

File: README.md (hunk 1 above)
Comment (jacknagz) [reply README.md]: Good point, I can combine them

File: README.md [hunk 3]
```diff
@@ -20,23 +21,32 @@
...
 ---
 
-## Why Panther?
+## About Us
 
-Panther is developed by a [dedicated team](https://runpanther.io/about/) of cloud security practitioners and is designed to be:
+We are a San Francisco based [startup](https://www.crunchbase.com/organization/panther-labs) comprised of security engineers who have spent years building large-scale detection and response for cloud-first companies such as Airbnb, Amazon, Riverbed, and more. Panther was founded by the core architect of [StreamAlert](https://github.com/airbnb/streamalert/), a project open-sourced at Airbnb to create a cloud-native solution for automated log analysis.
 
-- **Flexible:** Perform advanced analysis on log data and cloud infrastructure with [Python-based detections](https://github.com/panther-labs/panther-analysis)
+Panther is the next step in a journey of providing security teams with a cloud-native, scalable, modern alternative to traditional SIEMs. We designed Panther for massive scale, a rich and intuitive user experience, in-browser Python rule editors, first-class AWS support, and more.
```
[the remark this answers is not in the document]
Comment (jacknagz) [reply README.md]: Helpful! I made a slight iteration

File: README.md [hunk 4]
```diff
@@ -20,23 +21,24 @@
...
 
-- **Flexible:** Perform advanced analysis on log data and cloud infrastructure with [Python-based detections](https://github.com/panther-labs/panther-analysis)
-- **Automated:** Fast and simple deployments with AWS CloudFormation
-- **Scalable:** Built on Serverless technology for cost and operational efficiency at any scale
-- **Secure:** Least-privilege and encrypted infrastructure deployed into your cloud
-- **Integrated:** Analyze popular security logs, gather information about your cloud resources, and send alarms to commonly used destinations
+Panther is the next step in providing security teams with a modern alternative to traditional SIEMs. We designed Panther for massive scale, with a rich and intuitive user experience, in-browser Python rule editing, first-class AWS support, and more.
```
Comment (austinbyers) [inline README.md]: > Panther is the next step in providing security teams with a modern alternative to traditional SIEMs.

That still reads passive / tepid. Is that intentional? We're just the next step or we *are* the alternative?

> Panther provides security teams a modern alternative to traditional SIEMs.

> Panther is the next step for security teams who need a modern alternative to traditional SIEMs.

File: README.md (hunk 1 above)
Comment (austinbyers) [reply README.md]: I like it!

File: README.md [hunk 5]
```diff
@@ -20,23 +21,24 @@
...
+Panther is the destination for all security data to enable threat detection, compliance, historical search, and security investigations. Panther's core features include:
+
+- **[Log Analysis](https://runpanther.io/log-analysis):** Real-time detection of suspicious activity with [Python rules](https://github.com/panther-labs/panther-analysis/tree/master/analysis/rules)
+- **[Compliance](https://runpanther.io/compliance/):** Real-time monitoring and enforcement of AWS infrastructure best practices with [Python policies](https://github.com/panther-labs/panther-analysis/tree/master/analysis/policies)
 - **Alerting:** Send notifications to your team when new issues are identified
-- **Automatic Remediation:** Correct insecure infrastructure as soon as new issues are identified
+- **Automatic Remediation:** Correct insecure infrastructure as soon as possible
+
+Panther is designed to provide flexible detection logic, secure and scalable infrastructure run within your AWS cloud, support for commonly security tooling, and automation for painless deployments.
```
Comment (austinbyers) [inline README.md]: Typo: "support for commonly security tooling"

I'm guessing it was intended to be "commonly used security tooling," which I would say reads cleaner as just "common security tooling"

Comment (jacknagz) [reply README.md]: Yep yep, small typo. I was playing with the phrasing and forgot to adjust that word

File: README.md (hunk 4 above)
Comment (jacknagz) [reply README.md]: I was more so just speaking on the journey/evolution. I like the second suggestion one a lot!

**Pull request merged**